src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -44,48 +44,48 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         StringBuilder strBuf = new StringBuilder();  
         
-        HttpURLConnection conn=null;
-        BufferedReader reader=null;
+        HttpURLConnection conn = null;
+        BufferedReader reader = null;
 
         try {  
-            //Declare the connection to civic information API url
-            URL url = new URL(baseURL + "AIzaSyBlNN_uUrbqY2uRaBpKTD-5Jbw6nk8n0_k");  
+            URL url = new URL(baseURL + "AIzaSyBlNN_uUrbqY2uRaBpKTD-5Jbw6nk8n0_k");
             conn = (HttpURLConnection) url.openConnection();  
 
             conn.setRequestMethod("GET");
             conn.setRequestProperty("Accept", "application/json");
             
-            if (conn.getResponseCode() != 200) {
+            if (conn.getResponseCode() != HttpURLConnection.HTTP_OK) {
                 throw new RuntimeException("HTTP GET Request Failed with Error code : "
```

Comment:
Update this exception if you end up changing the method to PUT instead of GET. Ignore otherwise.